steering_vectors/aggregators.py

```diff
@@ -1,10 +1,33 @@
-from typing import Callable
+from typing import Callable, Union
 
+from sklearn.linear_model import LinearRegression, LogisticRegression
 from torch import Tensor
 import torch
+import torch.nn.functional as F
 
 
 Aggregator = Callable[[Tensor, Tensor], Tensor]
+Regression = Union[LinearRegression, LogisticRegression]
```

Comment:
Nit: I would remove this if it's not used anywhere

Reply:
Resolved by [7ba358f](https://github.com/steering-vectors/steering-vectors/pull/22/commits/7ba358f8229cbe662932a7d35cffe92cefbf6e28)